SCClassLibrary/Common/GUI/Base/QView.sc

```diff
@@ -89,7 +89,7 @@ View : QObject {
 		this.background = color;
 	}
 
-	backgroundImage_ { arg image; this.setBackgroundImage(image) }
+	backgroundImage_ { arg image, tileMode=1, alpha=1.0, fromRect; this.setBackgroundImage(image, tileMode, alpha, fromRect) }
```

Comment:
I agree with @scztt here. We generally avoid adding extra arguments to `variable_` setter methods. Users who need the extra arguments should use `setBackgroundImage`.

Reply:
@jamshark70, @scztt, thanks for the feedback !

Does it mean that issue #2575 should be rejected ?
And that we should rewrite this part of the documentation to use setBackgroundImage ? (this example is in `Image.schelp` today, but it was requested to move it to `view.schelp` on #3702)

```supercollider
(
    a = Image.new(SCDoc.helpSourceDir +/+ "images/Swamp.png");
    w = Window.new("Image background");
    l = Slider2D.new(w, Rect(10,10,200,200))
        .backgroundImage_( a, 5, 1, Rect(0,0,10,10) );
    w.front;
    a.free; // safe
)
```

Reply:
@scztt / @jamshark70 where are we on this?

Reply:
I've just fixed that, I reverted the changes to the setter and uses setBackgroundImage in the example.